snapcraft.yaml

```diff
@@ -9,6 +9,9 @@ description: |
   - manage updates to the system.
   - control the state of other snappy packages.
   - browse the store to install new snappy packages.
+
+  Please find the source code for this snap at:
+  https://code.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git/snapweb
```

Comment:
This should be github.com/snapcore/snapweb as the LP one is just a step to get this repository plugged into the release process but not the real origin of the snap.